hyperspy/drawing/mpl_he.py

```diff
@@ -28,6 +28,7 @@ class MPL_HyperExplorer(object):
     """
 
     def __init__(self):
+        self.scalebar_color = 'white'
```

Comment:
This line is needed in order to provide a field that can be set from `signal.plot()`